custom_components/healthchecksio/binary_sensor.py

```
@@ -39,10 +53,23 @@
-    def __init__(self, hass, check_data, config_entry):
+    coordinator: HealthchecksioDataUpdateCoordinator
+
+    _attr_attribution = ATTRIBUTION
+    _attr_device_class = BinarySensorDeviceClass.CONNECTIVITY
+
+    def __init__(
+        self,
+        *,
+        hass: HomeAssistant,
+        coordinator: HealthchecksioDataUpdateCoordinator,
+        ping_url: str | None = None,
```

Ruff (0.11.9)

Unused method argument: `ping_url`

(ARG002)